assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview.js

```diff
@@ -261,7 +275,10 @@ const Overview = ( {
 						<Cell { ...quarterCellProps }>
 							<DataBlock
 								stat={ 2 }
-								className="googlesitekit-data-block--visitors googlesitekit-data-block--button-3"
+								className={ classnames(
+									'googlesitekit-data-block--visitors googlesitekit-data-block--button-3',
+									topAlignedDataBlockClass
```

Comment:
Even though this wasn't mentioned in the IB or the issue, I think this wasn't found in the initial discovery. When the Goals CTA is shown, if the data blocks aren't top aligned, the layout breaks in smaller screens. Example [here](https://google.github.io/site-kit-wp/storybook/main/?path=/story/modules-searchconsole-widgets-searchfunnelwidget--ready-with-create-goal-cta).